hw05.py

```diff
@@ -0,0 +1,22 @@
+# hw04.py -path_csv example.csv -col_name Marker_Strategy
+import argparse
+import csv
+import json
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-csv')
+parser.add_argument('-json')
+ns = parser.parse_args(sys.argv[1:])
+
+
+data = {}
+with open(ns.csv) as csvFile:
```

Comment:
It'd be better to add the mode in which the file is opened:
`with open(ns.csv, 'r') as csvFile:`